Tasks/FileTransform/task.json

```diff
@@ -0,0 +1,72 @@
+{
+    "id": "8ce97e91-56cc-4743-bfab-9a9315be5f27",
+    "name": "FileTransform",
+    "friendlyName": "File Transform",
+    "description": "File Transform",
+    "author": "Microsoft Corporation",
+    "helpMarkDown": "File Transforms & Variable Substitution",
```

Comment:
Get the description, friendly name, helpmarkdown from the PM.

Reply:
sure